Plugins/Flow.Launcher.Plugin.PluginsManager/PluginsManager.cs

```diff
@@ -292,10 +293,87 @@ await Http.DownloadAsync(x.PluginNewUserPlugin.UrlDownload, downloadToFilePath)
                         ContextData =
                             new UserPlugin
                             {
-                                Website = x.PluginNewUserPlugin.Website, UrlSourceCode = x.PluginNewUserPlugin.UrlSourceCode
+                                Website = x.PluginNewUserPlugin.Website,
+                                UrlSourceCode = x.PluginNewUserPlugin.UrlSourceCode
                             }
                     });
 
+            if (resultsForUpdate.Count() > 1)
+            {
+                var updateAllResult = new Result
+                {
+                    Title = Context.API.GetTranslation("plugin_pluginsmanager_update_all_title"),
```

Comment:
Currently `pm update all plugins` is displayed if more than 2 updates are available. How can I change this so that only `pm update` but the translation is still displayed as the title of the result?

Reply:
Nope, the original idea was just to call `pm update` and show the option to run all updates to avoid introducing a new argument/command

Reply:
>the translation is still displayed as the title of the result?

Sorry I am also not quite understanding this line. Are you after a translated result title?

Reply:
Currently it look like this: ![Screenshot 2023-11-28 185152](https://github.com/Flow-Launcher/Flow.Launcher/assets/93255373/619cc0ce-ea2e-41a7-8971-521d20053147)
But as written in https://github.com/Flow-Launcher/Flow.Launcher/issues/980#issuecomment-1014272510 it should look like this:
![Screenshot 2023-11-28 185040](https://github.com/Flow-Launcher/Flow.Launcher/assets/93255373/a8d505fa-02b7-4763-a760-c2172c922ad2)
So the title should only be displayed in the result, but not in the search bar, if this is possible at all :)

Reply:
> Currently it look like this: ![Screenshot 2023-11-28 185152](https://private-user-images.githubusercontent.com/93255373/286353795-619cc0ce-ea2e-41a7-8971-521d20053147.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDEzMjEzNDAsIm5iZiI6MTcwMTMyMTA0MCwicGF0aCI6Ii85MzI1NTM3My8yODYzNTM3OTUtNjE5Y2MwY2UtZWEyZS00MWE3LTg5NzEtNTIxZDIwMDUzMTQ3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTMwVDA1MTA0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRjYmNlYTdiMjhhNzkwZjM0ZWE5ZWU3NDMxNzJkMWJjMTE3MzA5Y2JkNjM2MzVjZjEwZjdiYzQ2ZTY3ZDhiZWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.egJex7BoIRfllWh5nMGiMdrNu-yGZoPx7az2-Bd2rZY) But as written in [#980 (comment)](https://github.com/Flow-Launcher/Flow.Launcher/issues/980#issuecomment-1014272510) it should look like this: ![Screenshot 2023-11-28 185040](https://private-user-images.githubusercontent.com/93255373/286354935-a8d505fa-02b7-4763-a760-c2172c922ad2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDEzMjEzNDAsIm5iZiI6MTcwMTMyMTA0MCwicGF0aCI6Ii85MzI1NTM3My8yODYzNTQ5MzUtYThkNTA1ZmEtMDJiNy00NzYzLWE3NjAtYzIxNzJjOTIyYWQyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTMwVDA1MTA0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNkY2I1ZDU2ZmFjZmQ4YTM1YzdiNzhjMzI1NmE2NGU0ODEzODAwZWM4OTk2MDZkOGJiZDg0Nzg4ODhiZDNhMGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.OZutimy37nSD9Y78zt_Sx_xiBvbj9rnMvzgeBOu1K2E) So the title should only be displayed in the result, but not in the search bar, if this is possible at all :)

that's ok, Its the autocomplete no need to worry